Source/SPUD/Private/SpudPropertyUtil.cpp

```diff
@@ -323,17 +324,43 @@ bool SpudPropertyUtil::VisitPersistentProperties(UObject* RootObject, const UStr
 		{
 			if (!IsBuiltInStructProperty(SProp))
 			{
+				const UScriptStruct* StructDefinition = nullptr;
+				void* StructPtr = nullptr;
+
+				// Check if it's a InstancedStruct
+             	if(SProp->Struct->GetName() == "InstancedStruct")
```

Comment:
I'm not keen on the string test here, can we use something like `GetStructTypeHash` instead?

Reply:
String testing is indeed not a good idea. Now I use IsChildOf function to check if the structure is FInstancedStruct.

Reply:
Also, Sorry I didn't notice SpudTest, I've added a test for this case.